bitcoin/src/blockdata/weight.rs

```diff
@@ -54,8 +54,13 @@ impl Weight {
         vb.checked_mul(Self::WITNESS_SCALE_FACTOR).map(Weight::from_wu)
     }
 
-    /// Constructs `Weight` from virtual bytes in const context.
-    pub const fn from_vb_const(vb: u64) -> Weight {
+    /// Constructs `Weight` from virtual bytes returning the unwrapped value.
+    ///
+    /// # Panics
+    ///
+    /// If the conversion from Virtual bytes overflows.
```

Comment:
In 574efa798abed08145d7285a32305ba1316b1614:

nit: `virtual` should not be capitalized